test/e2e/locators/modal.ts

```diff
@@ -101,4 +113,18 @@ export default class ModalLocators {
     return modal.locator('.bibtex-download-btn');
   }
 
+  // --------- recordset select related functions ------------ //
+
+  static getModalRows(modal: Locator): Locator {
```

Comment:
I just realized that you added this. Let's not do this and instead use the `RecordsetLocators.getRows` with proper container passed to it like [here](https://github.com/informatics-isi-edu/chaise/blob/master/test/e2e/utils/record-utils.ts#L581)